lib/Dns.php

```diff
@@ -190,6 +190,9 @@ public static function validate_input($rid, $zid, $type, &$content, &$name, &$pr
 
             case "L64": // TODO: implement validation.
                 break;
+			
+	    case "LUA": // TODO: implement validation.
```

Comment:
The changes have nothing to do with the navbar. It would be better to create a separate PR.